### PR TITLE
Add files specifying which labels to run jenkins jobs on

### PR DESCRIPTION
**Pull request by vpodzime**
These branch-specific files define where to run the jenkins jobs
on by default and when running on exotic platforms too.

### CODE REVIEW

**Comment by vpodzime** [comment]
Example runs:

**Comment by vpodzime** [comment]
I cannot really integrate this with jenkins before it's merged because that would break the jenkins jobs for other PRs and branches that don't have these files. Also, this needs to be backported to 3.7.x and 3.10.x.

File: build-scripts/get_labels_expr.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+from __future__ import print_function
+
+import argparse
+import sys
+import re
+
+COMMENT_OUT = re.compile(r"([^#]*).*")
+
+def get_args():
+    ap = argparse.ArgumentParser(description="Tool to a the Groovy expression specifying the " +
```
Comment (olehermanse) [inline build-scripts/get_labels_expr.py]: Incomplete sentence, did you mean `add`?

Comment (vpodzime) [reply build-scripts/get_labels_expr.py]: Ooops, fixed.

**Comment by vpodzime** [comment]
> Also, this needs to be backported to 3.7.x and 3.10.x.

And 3.12.x, of course.

**Pull request merged**